test/jdk/java/awt/Mixing/AWT_Mixing/OverlappingTestBase.java

```diff
@@ -376,7 +376,9 @@ protected void clickAndBlink(Robot robot, Point lLoc) {
     protected String failMessage = "The LW component did not received the click.";
 
     private static boolean isValidForPixelCheck(Component component) {
-        if ((component instanceof java.awt.Scrollbar) || isMac && (component instanceof java.awt.Button)) {
+        if (component == null ||
+                (component instanceof java.awt.Scrollbar) ||
+                isMac && (component instanceof java.awt.Button)) {
             return false;
         }
         return true;
```

Comment:
```suggestion
        return !(component == null ||
                (component instanceof java.awt.Scrollbar) ||
                (isMac && (component instanceof java.awt.Button)));
```
There's no need for the `if` statement, just return the value of the boolean expression. You can propagate the negation into the boolean expression. I suggest using parentheses to group the condition with `isMac` to unambiguously express the intention.

I prefer operators on the wrapped line, this is the way Java Coding Style suggests, and such a way emphasises it's a continuation line rather than a new statement.